1.7/administration/monitoring/performance-monitoring.md

```diff
@@ -0,0 +1,129 @@
+---
+post_title: Performance Monitoring
+menu_order: 1
+---
+<!-- https://github.com/dcos/dcos-docs -->
+
+Here are some recommendations for monitoring a DC/OS cluster. You can use any monitoring tool with these recommendations. The endpoints listed below will help troubleshoot when issues occur.
+
+Your monitoring tools should leverage historic data points so that you can track changes and deviations. You should monitor your cluster when it is known to be in a healthy state as well as unhealthy. This will give you a baseline for what is “normal” in the DC/OS environment. With this historical data, you can fine tune your tools and set appropriate thresholds and conditions. When these thresholds are exceeded, you can send alerts to administrators.
+
+Mesos and Marathon have the metrics types gauge and counter.
```

Comment:
<nitpicky>
1) Use "expose" or "provide" instead of "have" for metrics.

2) Sentence structure not very clear: 
     Mesos and Marathon expose the metrics types: gauge and counter.
     OR
     Mesos and Marathon expose the following(OR two) types of metrics:
     OR
     Mesos and Marathon expose two types of metrics: gauge and counter
</nitpicky>